### ISSUE TITLE
Launch JSON Config

### ISSUE THREAD

**Issue by nyteshade** [BartmanAbyss/vscode-amiga-debug#38]
So, I am guessing from your configuration that we are supposed to add a `"config"` JSON property that matches `configurations[x].config` for one of the configuration block objects in the `launch.json` file. <strike>However, the only way I can get it to do anything other than choose the first configuration is to make something else the first configuration manually.</strike> _I can only specify a newer kickstart. Nothing I do can get it to use the A1200 or A4000 configs._ 

Also, the A4000 configuration is not setup in the version of your extension pulled from the Visual Studio Code extension search. Am I doing something wrong? I'd like to develop some apps, not games, using this extension and they are aimed at "newer" Amigas running 3.1.4 and so on. 

Is there a way to tweak the WinUAE settings applied to each config?

**Comment by nyteshade** [comment]
Looking through the typescript code, I can see it reading the generated `default.uae` file from `debugAdapter.js`. However no changes I make here do anything. I suspect the extension uses the minified client.bundle.js file. What is the best way to regenerate the file for this configuration?

**Comment by BartmanAbyss** [comment]
Hey,
in the JSON you can use `A500`, `A1200`, `A3000` or `A4000`. This activates quickconfig in WinUAE. Just specify your kickstart (1.3 for A500, 3.0+ for the rest). When your program is launched, you can then hit F12 to go into the WinUAE settings, and adjust your screenmodes, etc. Then save the config, overwriting the generated default.uae

**Comment by nyteshade** [comment]
@BartmanAbyss so it seems that hitting F5 regenerates the default.uae file every time. Values saved do not persist and specifying "config": "A4000" in launch.json seems to be entirely ignored as it still uses the "A500" configuration. Is this a by product of the 1.0.0 version found in the extension search for Visual Studio Code?

**Comment by BartmanAbyss** [comment]
Hmm.. it should work in 1.0, but I‘m not sure. Try downloading the latest 1.1 preview from ‚releases‘ on GitHub.

**Comment by nyteshade** [comment]
Okay, so found another way to do it. I installed the latest 1.1 preview via vsix file and made the changes recommended in the CHANGELOG, including creating the amiga.json file. This still did not honor a "config" property in launch.json but then I noticed this little button at the bottom of the editor. Tapping on it gave me the choices I wanted.

![aha!](https://user-images.githubusercontent.com/225558/93386652-e308d100-f81c-11ea-973e-4ad1e292c710.png)
